src/modules/EventList/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ReactElement, MouseEventHandler, useCallback } from 'react';
+import React, { Fragment, ReactElement, MouseEventHandler, useCallback } from 'react';
```

Comment:
Instead of `<Frament></Fragment>` you can acatully just do `<></>` and not have to import anything anymore.